### DISCUSSION TITLE
ms-python linting not working

### DISCUSSION THREAD

**Discussion by fark0860** [VSCodium/vscodium#1632] [Q&A]
hello,
Recently I noticed that linting in python was not working as before 
previously things line .linspace(0,2,10)---> linspace(0,  2,  10)  automatically on saving the file 
This is not working anymore. Furthermore running linter from command pallete gives nothing 
What I have tried
disabling all extensions 
reinstalling the extension 
installing pylint extension 
restarting pc and codium
Additional info
codium ver 1.79.0
OS Linux UBUNTU 20.04
Any help is much appreciated

**Comment by nassimaugsburg** [comment]
I have the same issue on windows.

**Comment by daiyam** [comment]
Have you tried https://open-vsx.org/extension/detachhead/basedpyright? Thx
